firestore-bigquery-export/extension.yaml

```diff
@@ -461,6 +461,27 @@ params:
     validationErrorMessage: Please select an integer between 1 and 10
     default: 3
 
+  - param: LOG_LEVEL
+    label: Log level
+    description: >-
+      The log level for the extension. The log level controls the verbosity of
+      the extension's logs. The available log levels are: debug, info, warn, and
+      error. To reduce the volume of logs, use a log level of warn or error.
+    type: select
+    options:
+      - label: Debug
+        value: debug
+      - label: Info
+        value: info
+      - label: Warn
+        value: warn
+      - label: Error
+        value: error
+      - label: Silent
+        value: silent
+    default: info
```

Comment:
not sure what the default should be, as it seems too verbose at the moment. info seems the right level to be default, maybe we need to downgrade some of the logs to debug (im thinking the logs in the package where they log "inserting" and "inserted"